src/modules/admin/components/Permissions/UserPermissions.tsx

```diff
@@ -2,17 +2,23 @@ import React from 'react';
 import { ColonyRole } from '@colony/colony-js';
 import PermissionsLabel from '~core/PermissionsLabel';
 import { permissionsObject } from '~core/PermissionsLabel/permissions';
+import { getMainClasses } from '~utils/css';
 
 import styles from './UserPermissions.css';
 
+interface Appearance {
+  padding: 'none';
+}
+
 interface Props {
   roles: ColonyRole[];
   directRoles: ColonyRole[];
+  appearance?: Appearance;
 }
 
 const displayName = 'admin.Permissions.UserPermissions';
 
-const UserPermissions = ({ roles, directRoles }: Props) => {
+const UserPermissions = ({ roles, directRoles, appearance }: Props) => {
```

Comment:
Why is this component still in `admin`, did I forgot to mention to move it? Or maybe it was a bad rebase?

Anyway, can you please move it out of `admin` and into `dashboard` ?